packages/plugin/tests/require-type-pattern-with-oneof.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { GraphQLRuleTester } from '../src';
+import { rule } from '../src/rules/require-type-pattern-with-oneof';
+
+const ruleTester = new GraphQLRuleTester();
+
+ruleTester.runGraphQLTests('require-type-pattern-with-oneof', rule, {
+  valid: [
+    /* GraphQL */ `
+      type T @oneOf {
+        ok: Ok
+        error: Error
+      }
```

Comment:
should I requires schema and validate that `type T` is used as the result type for some mutation field?

Reply:
We don't need that at the moment, mainly because `@oneOf` might be used for other things, not just mutations